torchtext/csrc/CMakeLists.txt

```diff
@@ -1,9 +1,78 @@
+if (MSVC)
+  set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
Can you add a comment that this approach has a limitation, with the reference to https://github.com/pytorch/pytorch/pull/3650?